udata_front/theme/gouvfr/assets/js/components/search/search.vue

```diff
@@ -154,16 +154,46 @@
                 :changePage="changePage"
                 class="fr-mt-2w"
               />
+            <div v-else>
+              <ActionCard
+                :title="$t(`Didn't find what you are looking for ?`)"
+                :icon="search2Icon"
+                type="primary"
+                >
+                <p class="fr-m-0 fr-text--sm">
+                  {{ $t("Try to reset filters to widen your search.") }}<br/>
+                  {{ $t("You can also give us more details with our feedback form.") }}
+                </p>
+                  <template v-slot:actions>
+                    <button @click="reloadForm" class="fr-btn fr-btn--secondary">
```

Comment:
Somehow when clicking the `reloadForm`, the URL's params get updated with weird `isTrusted=true&_vts=XXXXx`?